angular.json

```diff
@@ -50,7 +43,8 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "api-collab:build"
+            "browserTarget": "api-collab:build",
+            "proxyConfig": "proxy.conf.json"
```

Comment:
any reason to add this "proxyConfig" ?

Reply:
I was following the instructions from 
https://github.com/angular/angular-cli/blob/master/docs/documentation/stories/proxy.md